graph/src/data/graphql/effort.rs

```diff
@@ -381,26 +381,52 @@ impl LoadManager {
         overloaded: bool,
         wait_ms: Duration,
     ) -> f64 {
+        // The rates by which we increase and decrease the `kill_rate`; when
+        // we increase the `kill_rate`, we do that in a way so that we do drop
+        // fewer queries as the `kill_rate` approaches 1.0. After `n`
+        // consecutive steps of increasing the `kill_rate`, it will
+        // be `1 - (1-KILL_RATE_STEP_UP)^n`
+        //
+        // When we step down, we do that in fixed size steps to move away from
+        // dropping queries fairly quickly so that after `n` steps of reducing
+        // the `kill_rate`, it is at most `1 - n * KILL_RATE_STEP_DOWN`
+        //
+        // The idea behind this is that we want to be conservative when we drop
+        // queries, but aggressive when we reduce the amount of queries we drop
+        // to disrupt traffic for as little as possible.
+        const KILL_RATE_STEP_UP: f64 = 0.1;
+        const KILL_RATE_STEP_DOWN: f64 = 2.0 * KILL_RATE_STEP_UP;
+
+        lazy_static! {
+            static ref KILL_RATE_UPDATE_INTERVAL: Duration = Duration::from_millis(1000);
+        }
```

Comment:
This doesn't need `lazy_static`, I believe it can be an ordinary constant.

Reply:
True; fixed that.